shard.yml

```diff
@@ -61,7 +61,7 @@ dependencies:
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
-    version: ~> 1.1.0
+    version: ~> 1.2.0
```

Comment:
I had to update ameba to get it to run locally for some reason...

Reply:
reverted because it tanks on the CI... :thinking: so I either get to run it locally, or on CI, but not both...